website/thaliawebsite/settings/production.py

```diff
@@ -35,9 +35,9 @@
     "default": {
         "ENGINE": "django.db.backends.postgresql",
         "USER": os.environ.get("POSTGRES_USER", "postgres"),
-        "PASSWORD": os.environ.get("POSTGRES_PASSWORD", ""),
+        "PASSWORD": os.environ.get("POSTGRES_PASSWORD", None),
         "NAME": os.environ.get("POSTGRES_DB"),
-        "HOST": os.environ.get("DJANGO_POSTGRES_HOST"),
+        "HOST": os.environ.get("DJANGO_POSTGRES_HOST", ""),
```

Comment:
This could be set to `postgres` which is a reasonable assumption for the name of a Postgres Docker host. However an empty string is fine too, given that that is the [default](https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-HOST) anyway.

Reply:
I specifically choose the empty host because of this part
>If you’re using PostgreSQL, by default (empty HOST), the connection to the database is done through UNIX domain sockets (‘local’ lines in pg_hba.conf).

This means that local authorization is easier when not running on docker. I think this is a good default, as the name of the postgres host in docker systems can differ.